fleetctl/fleetctl.go

```diff
@@ -130,6 +133,9 @@ func init() {
 	globalFlagset.StringVar(&globalFlags.CertFile, "cert-file", "", "Location of TLS cert file used to secure communication with the fleet API or etcd")
 	globalFlagset.StringVar(&globalFlags.CAFile, "ca-file", "", "Location of TLS CA file used to secure communication with the fleet API or etcd")
 
+	globalFlagset.StringVar(&globalFlags.EtcdUsername, "etcd-username", "", "Username used to authenticate with etcd")
+	globalFlagset.StringVar(&globalFlags.EtcdPassword, "etcd-password", "", "Password used to authenticate with etcd")
+
```

Comment:
Thanks for the documentation and the patch, but honestly I don't like the fact that username and password will show up on the cmdline where every process will just grab them... so could you please make just a one option where it reads the input from the config file ?

Thank you!